src/app/core/theme.component.ts

```diff
@@ -33,14 +42,57 @@ import { EventSurveyComponent } from '../features/event/event-survey/event-surve
     <h2>Formularz zapisu dla visitora</h2>
     <app-visitor-form></app-visitor-form>
 
-    <h2>Widok ankiety dla usera</h2>
+    <h2>Lista uczestników eventu</h2>
```

Comment:
Tutaj chyba powinna być klasa dla zmiany czcionki fontu: `font-heading-2`

Reply:
Czy aby na pewno musimy się o to martwić w theme, jak to jest tylko informacja dla nas co to za komponent?